libpod/kube.go

```diff
@@ -0,0 +1,276 @@
+package libpod
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/containers/libpod/pkg/lookup"
+	"github.com/containers/libpod/pkg/rootless"
+	"github.com/containers/libpod/pkg/util"
+	"github.com/cri-o/ocicni/pkg/ocicni"
+	"github.com/ghodss/yaml"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	KubeNotSupportedError = errors.New("This libpod function is not supported in kube translation yet")
+)
+
+// InpectforKube takes a slice of libpod containers and generates
+// one v1.Pod description that includes just a single container.
+func InspectForKube(containers []*Container) error {
```

Comment:
Make this work on a single container and return a string (the marshalled yaml)

Reply:
We can do the iterating part and output in `podman inspect`

Reply:
(We may want to return the actual Kube YAML structure instead of a string at some point, but I don't really want to deal with the struct directly in the API right now)

Reply:
fixed up, yup